README.md

```diff
@@ -96,3 +96,23 @@ for url in ctadata.list_dir("cta/DL1/20241114/v0.1"):
         print("found keys", h5py.File(url.split("/")[-1]).keys())
 ```
 
+# Webdav Client
+
+To offer a smoother usage, `ctadata` offers a `webdav4` client, see [webdav4](https://github.com/skshetry/webdav4) for documentation.
```

Comment:
I am not sure if it's smoother, but anyway it's better to avoid disputable style qualifiers and say directly what is different about this alternative.

```suggestion
In order to make use of bare WebDAV interface of the storage, `ctadata` also provides a configured `webdav4` client (see [webdav4](https://github.com/skshetry/webdav4) for documentation).
```